src/modules/expense/Form.tsx

```diff
@@ -26,6 +25,37 @@ import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { DatePicker } from '@mui/x-date-pickers/DatePicker';
 import dayjs, { Dayjs } from 'dayjs';
 import { Regular1 } from '../../shared-components/Typography/Regular1';
+import { useSelector, useDispatch } from 'react-redux';
+//import { Regular1 } from '../../shared-components/Regular1';
+import {
+  setCategory,
+  description,
+  wallet,
+  frequency,
+  setSelectedEndAfter,
+  setDate,
+} from '../expense/expenseFormSlice';
+import { Category } from 'src/types/Category';
+import { PaymentFrequency } from 'src/types/PaymentFrequency';
+
+interface RootState {
+  newExpense: {
+    category: Category[];
+    textValue: string;
+    selectedWallet: string;
+    showAttachmentComponent: boolean;
```

Comment:
@Saffet01 values that do not belong to a new expense should not be in the Redux store (e.g. showAttachmentComponent, showContinueButton, switchChecked, toggleContinueButton, showPopup, showRepeatDetails, showSummarizeSection, repeatSectionContinueButton) should be removed.

Then, make sure to name values correctly:
textValue -> description
selectedWallet -> wallet
selectedFrequency -> frequency